openreview/openreview.py

```diff
@@ -256,36 +256,16 @@ def get_invitations(self, id = None, invitee = None, replytoNote = None, replyFo
         invitations.sort(key = lambda x: x.id)
         return invitations
 
-    def get_notes(self, id = None, paperhash = None, forum = None, invitation = None, replyto = None, tauthor = None, signature = None, writer = None, includeTrash = None, number = None, limit = None, offset = None, mintcdate = None):
-        """Returns a list of Note objects based on the filters provided."""
-        params = {}
-        if id != None:
-            params['id'] = id
-        if paperhash != None:
-            params['paperhash'] = paperhash
-        if forum != None:
-            params['forum'] = forum
-        if invitation != None:
-            params['invitation'] = invitation
-        if replyto != None:
-            params['replyto'] = replyto
-        if tauthor != None:
-            params['tauthor'] = tauthor
-        if signature != None:
-            params['signature'] = signature
-        if writer != None:
-            params['writer'] = writer
-        if includeTrash == True:
-            params['trash']=True
-        if number != None:
-            params['number'] = number
-        if limit != None:
-            params['limit'] = limit
-        if offset != None:
-            params['offset'] = offset
-        if mintcdate != None:
-            params['mintcdate'] = mintcdate
-
+    def get_notes(self, **params):
+        '''
```

Comment:
how will we know which parameters are ok for the API?

look into this - how does a user access the docstring?